learn/app-development/widgets/chart/chart-widget.md

```diff
@@ -230,6 +230,24 @@ Page.chart4_1Beforerender = function(widget, chartInstance) {
 
 [![](/learn/assets/onbeforerender-code.png)](/learn/assets/onbeforerender-code.png)
 
+### Customizing charts with rounded corners
+Inorder to achieve rounded corners for column and bar charts add class **'rounded-cornered-chart'** and for area chart add class **'rounded-area-chart'** to the chart widget.
+
+#### Column Chart
+[![](/learn/assets/rounded-column-chart.png)](/learn/assets/rounded-column-chart.png)
+
+#### Bar Chart
+[![](/learn/assets/rounded-bar-chart.png)](/learn/assets/rounded-bar-chart.png)
+
+### Customizing grid line on the charts
+Solid stroked line on x-axis and y-axis zero line can be achieved by adding **'stroked-zero-axis-line'** class

```

Comment:
Are these CSS classes available in nvd3? IF so, can we just show a list all available CSS classes and the effect adding them will have?

Reply:
These are the classes written by us only on the platform using nvd3 selectors, like below

.stroked-zero-axis-line {
      .nvd3 .nv-axis .zero line {
        stroke: rgba(24, 21, 21, 1);
        stroke-dasharray: none; 
      }
    }